### PR TITLE
[v2] Added support for loading .env files

### LINKED ISSUE #102: [v2] Add support for `.env` files
Add support for loading environment variables via `.env` files in the `v2-dev` branch.

### PR DESCRIPTION
**Pull request by harshkhandeparkar**
Closes #102 

#### Changes
- Added support for loading .env files using `godotenv` library.
- Gitignored `.env` and `*.db` files.

### CODE REVIEW

File: cmd/backend.go [hunk 1]
```diff
@@ -17,6 +18,16 @@ func main() {
 	zerolog.SetGlobalLevel(zerolog.InfoLevel)
 	log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stderr})
 
+	// Load environment variables via .env files
+	log.Info().Msg("Attempting to load .env file.")
+	dotenv_err := godotenv.Load()
```
Comment (rajivharlalka) [inline cmd/backend.go:23]: How about adding a support for passing a env file name as a cli parameter. would help in switching environments easily without changing the actual env file but just the command that runs the binary.

**Review by rajivharlalka — CHANGES_REQUESTED** [review]
Also can you add an example env file, would be  better to have an example of what the parameters look like, with ig degault values of the varaibles. 

I would prefer a YAML format too for the env, looks doesnt need any changes([ref](https://github.com/joho/godotenv#:~:text=Or%20finally%20you%20can%20do%20YAML(ish)%20style)), but you can take a decision on that.

**Comment by harshkhandeparkar** [comment]
> Also can you add an example env file, would be better to have an example of what the parameters look like, with ig degault values of the varaibles.
> 
> I would prefer a YAML format too for the env, looks doesnt need any changes([ref](https://github.com/joho/godotenv#:~:text=Or%20finally%20you%20can%20do%20YAML(ish)%20style)), but you can take a decision on that.

I believe the YAML style is supported by default. I will add an empty `.env.template` file and will populate it and document the variables in #105 since this branch doesn't use any variables.

File: cmd/backend.go (hunk 1 above)
Comment (harshkhandeparkar) [reply cmd/backend.go]: Done

**Review by rajivharlalka — APPROVED** [review]
LGTM

**Pull request merged**